otel/conf.go

```diff
@@ -0,0 +1,584 @@
+        operations:
+          # change data type from int64 -> double
+          - action: toggle_scalar_data_type
+      # system.swap.utilization -> swap/percent_used
+      - metric_name: system.swap.utilization
+        action: update
+        new_name: swap/percent_used
+      # duplicate swap/percent_used -> pagefile/percent_used
+      - metric_name: swap/percent_used
+        action: insert
+        new_name: pagefile/percent_used
+        operations:
+          # take sum over device dimension, retaining only state
+          - action: aggregate_labels
+            label_set: [ state ]
+            aggregation_type: sum
+      # system.swap.paging_ops -> swap/io
+      - metric_name: system.swap.paging_ops
+        action: update
+        new_name: swap/io
+        operations:
+          # delete single-valued type dimension, retaining only direction
+          - action: aggregate_labels
+            label_set: [ direction ]
+            aggregation_type: sum
+      # process.cpu.time -> processes/cpu_time
+      - metric_name: process.cpu.time
+        action: update
+        new_name: processes/cpu_time
+        operations:
+          # change data type from double -> int64
+          - action: toggle_scalar_data_type
+          # change label state -> user_or_syst
+          - action: update_label
+            label: state
+            new_label: user_or_syst
+            # change label value system -> syst
+            value_actions:
+              - value: system
+                new_value: syst
+      # process.disk.read_io (as named after custom split logic) -> processes/disk/read_bytes_count
+      - metric_name: process.disk.read_io
+        action: update
+        new_name: processes/disk/read_bytes_count
+      # process.disk.write_io (as named after custom split logic) -> processes/disk/write_bytes_count
+      - metric_name: process.disk.write_io
+        action: update
+        new_name: processes/disk/write_bytes_count
+      # process.memory.physical_usage -> processes/rss_usage
+      - metric_name: process.memory.physical_usage
+        action: update
+        new_name: processes/rss_usage
+        operations:
+          # change data type from int64 -> double
+          - action: toggle_scalar_data_type
+      # process.memory.virtual_usage -> processes/vm_usage
+      - metric_name: process.memory.virtual_usage
+        action: update
+        new_name: processes/vm_usage
+        operations:
+          # change data type from int64 -> double
+          - action: toggle_scalar_data_type
+
+  # filter to include only agent metrics supported by cloud monitoring
+  filter/agent:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - otelcol_process_uptime
+          - otelcol_process_memory_rss
+          - otelcol_grpc_io_client_completed_rpcs
+          - otelcol_googlecloudmonitoring_point_count
+
+  # convert from windows perf counter formats to cloud monitoring formats
```

Comment:
Is my understanding correct that the `metricstransform/iis` and `metricstransform/mssql` here define the conversion. But because we have not yet enabled the receivers for `iis` and `mssql` on purpose, these are effectively no-op?

Reply:
Yes, it will only be effective when it's used in the pipelines.